### PR TITLE
Update controller-runtime and controller-tools owners

### PR DESCRIPTION
**Pull request by sbueringer**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -264,28 +264,6 @@ aliases:
     - srm09
   # end sigs.k8s.io/cluster-api-provider-vsphere/OWNERS_ALIASES
 
-  # mostly copied from sigs.k8s.io/controller-runtime/OWNERS_ALIASES
```
Comment (sbueringer) [inline OWNERS_ALIASES:267]: I inlined the owner aliases as we're then able to update the approvers/reviewers ourselves without requiring top-level approval

File: config/jobs/kubernetes-sigs/controller-tools/OWNERS [hunk 2]
```diff
@@ -1,7 +1,12 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/devel/owners.md
+# Note: This is kept in sync with: https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES
```
Comment (listx) [inline config/jobs/kubernetes-sigs/controller-tools/OWNERS:2]: Can you add this to the commit message description? Or even better, explain in the commit message why we need to keep this in sync with https://github.com/kubernetes-sigs/controller-tools/blob/master/OWNERS_ALIASES? Is there any other context/history that could be useful here for future readers?

Comment (sbueringer) [reply config/jobs/kubernetes-sigs/controller-tools/OWNERS]: Added a note in both places. Basically the idea is that approvers/reviewers of the respective repositories have the same rights for the jobs corresponding to the repositories.

**Comment by listx** [comment]
/lgtm
/approve
/hold

Holding in case my approval goes through (not sure if you want others to look at this also).

**Pull request merged**